decorators.rb

```diff
@@ -0,0 +1,24 @@
+require './nameable'
```

Comment:
- [ ] Good job so far, :clap: However, Please ensure that you remove the `./` and use on `nameable` to target the project directory. 
- [ ] Kindly use `require_relative` as the most recommended way. :slightly_smiling_face: 
- Typical Example: :arrow_down: 
`require_relative 'nameable'`